packages/utils/src/index.js

```diff
@@ -201,18 +222,22 @@ export const captureSuspensePromises = executors => {
   return values;
 };
 
-//ruturns the length of all arrays in obj
-// { a: [1, 2], b: [1, 2, 3] } => 5
+/**
+ * Returns the length of all arrays in obj
+ * @param {*} obj An objects whose property values are arrays.
+ */
 export const countArrays = obj =>
   Object.values(obj).reduce((total, items) => total + (items ? items.length : 0), 0);
 
 /**
  * Compares two version strings or number arrays in the major.minor.patch format.
- * Returns true if comp if each element of comp is greater than than base.
+ * @param {Array<Number>|String} comp The version to compare.
+ * @param {Array<Number>|String} base The version against which to compare.
+ * @returns True if each element of comp is greater than or equal base.
  */
 export const versionGreaterOrEqualTo = (comp, base) => {
   const parseVersion = input => {
-    if (typeof input === 'object') {
+    if (input instanceof Array) {
```

Comment:
Tiny random change. When I originally wrote this function I didn't know about `instanceof` and I had intended this check to be for arrays.

Reply:
It's better to use `Array.isArray()`. instanceof with Arrays produce some problems ...

Reply:
Do Javascript’s quirks never end ☹️

Reply:
I can create PR. Because it's may produce problems for me.